Software_Carbon_Intensity/Software_Carbon_Intensity_Specification.md

```diff
@@ -123,77 +123,115 @@ The list should be maintained in alphabetic order. DELETE This Row
+***Embodied Emissions (Embodied) ***
+[placeholder]
 
-Carbon aware software will optimize the timing and location of operation to minimize emissions associated with operation. This could consist of moving computation to regions with cleaner grid emissions or delaying jobs to cleaner periods (or a combination of both). Energy efficient software will also run on hardware that requires less energy to operate or the software can be re-architected to require less energy to execute. All combined, these effects should be reflected in the total operational emissions. 
+Some categories of applications that apply Embodied Emissions (Embodied): 
+- **Hardware Efficient Applications:** Applications that use the least physical resources possible to perform their function.
 
+## Characteristics for calculating total carbon emissions, `C`
 
-#### Embodied Emissions
+#### C should be sensitive to carbon awareness, energy efficiency, or hardware efficiency improvements to the application
 
-[placeholder]
 
-#### Calculating total emissions, C
+#### Calculating Total Emissions, C
 
 Total emissions, C, is the sum of operational and embodied emissions.
 
 Overall, if we calculate the total carbon emissions C of an application as X, and then we make the application carbon aware, a characteristic of the methodology of calculation is that the value of X should go down. If we make the application more energy efficient the value of X should go down. If we make the application more hardware efficient, the value of X should go down.
 
 The challenges with existing methods of calculation is that they are only sensitive to some of the above methods of making an application green. For instance there are methods of calculation of carbon emissions that are purely cost based, using those methods results in a value of C that will not not change if you make the application carbon aware or energy efficient.
 
+To calculate the carbon intensity we need to compare total emissions (C) against a baseline (R).
+
+`R` = Baseline, the other measure we will be comparing against (baseline, or carbon counterfactual) 
+
+` Total Emissions = C / R`
+
 
 
 ## Exclusions
 
-### Carbon Offsets and Electricity Attribute Certificates
+### Market-Based Measures: Carbon Offsets and Electricity Attribute Certificates
+
+Because this standard lays out a consequential methodology for calculating the emissions associated with a piece of software, the following must not be included in the calculation: 
+- carbon offsets  
+- electricity attribute certificates (EACs) 
+- Power Purchase Agreements (PPAs)
+- Renewable Energy Credits (RECs)
+
+Market-based measures which includes companies agreements and infrastructure: We don’t need the market-based info to be shared by API to Developers, which keeps things simple, and prevents concerns about publishing ‘seemingly less green data about Azure’. This can be viewed in two ways:
```

Comment:
Do we need this discussion of market based in the standard? Should we just exclude?

Reply:
Agreed, I think we can limit to just this sentence "Market-based measures which include companies agreements and infrastructure".

I think a lot of this specific nuance we can explain with some worked out use cases to really bring it home which we can list at the **end of the spec**. It's going to need a lot of hand-holding to explain this to people. E.g. "Company X is 100% matched by renewables, but that doesn't matter since the spec clearly states in exclusions that renewable credits are not counted so ........"

Reply:
I believe we can keep it crisp, but add a definition. Propose two terms to disambiguate: 

1. 'market-based measures' 
2. Infrastructure measures

Should these live in the definitions section? 

We will __really__ want to be crisp here when discussing methodology with stakeholders at all tiers

Reply:
Would love review! 

Propose merging and disambiguating in separate issue/PR

Reply:
Yes let's have them in a separate issue and keep it crisp here. But the nuance is much needed so let's not lose that!